Plugins/Reqnroll.Verify/Reqnroll.Verify.ReqnrollPlugin/build/Reqnroll.Verify.props

```diff
@@ -1,5 +1,9 @@
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" TreatAsLocalProperty="TaskFolder;TaskAssembly">
-	<ItemGroup>
-		<ReqnrollGeneratorPlugins Include="$(_VerifyGeneratorPluginPath)" />
-	</ItemGroup>
+  <ItemGroup>
+    <None Include="$(_VerifyGeneratorPluginPath)" >
```

Comment:
This should be `$(_VerifyRuntimePluginPath)` instead of `$(_VerifyGeneratorPluginPath)`.